WooCommerce/Classes/ViewRelated/Orders/Order Creation/FlowCoordinator/NewSimplePaymentsNoticeViewModel.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+final class NewSimplePaymentsNoticeViewModel {
+
+    private let simplePaymentsNoticeView: BottomSheetNoticeView
+
+    /// Redirects to `HubMenu`tabBar
+    ///
+    private var completionHandler: (() -> Void)? = {
```

Comment:
I find the `completionHandler` naming a bit confusing here, there are two ways of completion here, dismissing by canceling and navigating to menu. This closure only takes the latter into account, so I would rename it to something more specific, like `navigateToMenuButtonWasTapped`

Reply:
Makes sense! Changed here: fdf13a636fd3bd800ba71ad302d42e0984d43981